### DISCUSSION TITLE
“Developer Mode” roadblock on iOS 16.1?

### DISCUSSION THREAD

**Discussion by Umanoff** [airsdk/Adobe-Runtime-Support#2271] [General]
Am I the only one to hit this snag? I’m in the middle of development of an iOS app (using AIR SDK 50). iOS 16.1 won’t run the app. 
iOS 16.0 DID run the app after enabling “Developer Mode”, but Apple’s instructions for enabling Developer Mode don’t seem to be correct for 16.1 
Apple’s regular tech support seems to know less about this than I do (they sent me this: https://developer.apple.com/documentation/xcode/enabling-developer-mode-on-a-device) and Apple’s developer support no longer answers the phone! 
The apple documentation, besides being wrong about Developer Mode on 16.1, is unclear to me about this. Apparently you can also turn on Developer Mode on any given device with Xcode (I haven’t tried this yet), but this is not a realistic solution to enabling software for the average user. 
Somewhere I read in Apple’s recent documentation that this Developer Mode issue should not be an obstacle for apps using Enterprise account certificates but that also seems to be wrong so far. 

I'd appreciate any insight on this. I'll post corrections or solutions if I find any.

**Comment by Umanoff** [comment]
Update on this. The new 16.2 Beta has reinstated Developer Mode.  
Just for the info. of anyone interested: Apple says that Developer Mode should not need to be enabled to run Enterprise account apps, but as of 16.2 it DOES need to be enabled.

**Reply by truonggiang0710** [reply]
@Umanoff 
If the phone is installed enterprise app before enabling developer mode => You can trust the app and start the app without issue.
If the phone is enabled in developer mode before installing the enterprise app => The app cannot verify. The phone must be rebooted, and the app can verify successfully.
This issue exists on iPhone 12 and above.
